src/Ryujinx.Graphics.Gpu/GpuContext.cs

```diff
@@ -219,6 +222,9 @@ public ulong GetTimestamp()
         {
             ulong ticks = ConvertNanosecondsToTicks((ulong)PerformanceCounter.ElapsedNanoseconds);
```

Comment:
```suggestion
            ulong ticks = ConvertNanosecondsToTicks((ulong)PerformanceCounter.ElapsedNanoseconds) - _firstTimestamp;
```
Why not do that here directly?

Reply:
Yes, you can keep it.